.github/ISSUE_TEMPLATE/bug_report_form.yml

```diff
@@ -0,0 +1,47 @@
+---
+name: Bug report
+description: Create a report to help us improve
+body:
+  - type: textarea
+    id: bug-description
+    attributes:
+      label: Describe the Bug
+      placeholder: A clear and concise description of what the bug is.
+    validations:
+      required: true
+  - type: textarea
+    id: bug-reproduction
+    attributes:
+      label: Steps to reproduce the behavior
+      placeholder: A list of steps to reproduce the described bug.
+    validations:
+      required: true
+  - type: dropdown
+    id: browser
+    attributes:
+      label: Browser
+      description: What browser are you using?
+      multiple: true
+      options:
```

Comment:
Thanks for your PR, but im not really sure about this one I dont think all people know that when they use opera or something that its a chrome based browser so maybe this should be a text input or if possible a dropdown where you can additionaly write something else if its not one of the two.

Reply:
Sure I can change it to a regular text input. I was just mirroring the style of the existing templates for these when I converted them.

Reply:
I think it was more meant of an example on the old version

Reply:
@mcdonnnj are you planning to do the change with the text input?

Reply:
@henrik9999 Ah, sorry this completely fell off my radar. Yes, I will update my branch tonight or tomorrow at the latest. Thanks for the bump!

Reply:
@henrik9999 Please see [06932e2](https://github.com/MALSync/MALSync/pull/1079/commits/06932e2add919ccaed5c15e5e68f34e77bfbf61c) and let me know if that satisfies your concerns.